frame/staking-rewards/src/lib.rs

```diff
@@ -271,8 +271,15 @@ pub mod pallet {
 			+ One;
 
 		type FinancialNft: nonfungibles::Mutate<AccountIdOf<Self>>
-			+ nonfungibles::Create<AccountIdOf<Self>>
-			+ FinancialNft<AccountIdOf<Self>>;
+			+ Create<
+				AccountIdOf<Self>,
+				ItemId = Self::FinancialNftInstanceId,
```

Comment:
I think we can remove `Self::FinancialNftInstanceId` altogether and just use T::FinancialNft::ItemId directly

Reply:
Yeah, remove it entirely; we can just use `T::FinancialNft::ItemId` as the `FinancialNftInstanceId`

Reply:
yeah makes sense, it removes some verbosity